### ISSUE TITLE
Alt attribute

### ISSUE THREAD

**Issue by jim-rosales** [Vdaria/prj-rev-bwfs-dasmoto#2]
When adding images to a site, it is best to include the [alt attribute](https://en.wikipedia.org/wiki/Alt_attribute) as well. The reason for this varies from improving search engine results via SEO, increasing the [accessibility](https://www.codecademy.com/articles/accessibility) of your site, and because it is used as a backup in case, the image cannot be loaded. 

For example, you can do the following:
`<img src="./images/hacksaw.jpg"/ alt="Hacksaw">`

https://github.com/Vdaria/prj-rev-bwfs-dasmoto/blob/c5bceb682f846b9de5320b7df3e8dfa8995e3894/DASMOTO'ARTS/index.html#L27
